### PR TITLE
HM2_task2

### PR DESCRIPTION
**Pull request by MichaelN98**
Add HM2 with solved all tasks

### CODE REVIEW

File: HM2_task2.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# 2
+Eleks = ['Mark', 'Anna', 'Diana', 'Alex', "Inga"]
+Toshiba = ['Mari', 'Vika', 'Alex', 'Mark', 'Olya']
+
+Toshiba.append(Eleks)
+print(Toshiba)
```
Comment (bohdandovbsyh) [inline HM2_task2.py:6]: eleks still have employees please fix

File: HM2_task3.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+casino_blacklist = ["Jack Pie", "Erik Man", "Lili Evans"]
+poker_blacklist = ["Patrick Low", "Ann Downstown", "Mari Darret"]
+alcohol_blacklist = ["Albert Second", "Kate Signi", "Iri Mane"]
+
+casino_blacklist.extend(alcohol_blacklist)
+casino_blacklist.extend(alcohol_blacklist)
+print(casino_blacklist)
+
+# second option
+# all_blacklists = list
```
Comment (bohdandovbsyh) [inline HM2_task3.py]: this line makes no sense

File: HM2_task3.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# 3
+
+casino_blacklist = ["Jack Pie", "Erik Man", "Lili Evans"]
+poker_blacklist = ["Patrick Low", "Ann Downstown", "Mari Darret"]
+alcohol_blacklist = ["Albert Second", "Kate Signi", "Iri Mane"]
+
+casino_blacklist.extend(alcohol_blacklist)
+casino_blacklist.extend(alcohol_blacklist)
+print(casino_blacklist)
```
Comment (bohdandovbsyh) [inline HM2_task3.py]: It is not a solution. You just add all lists together. Please read task more cerefully 
image

File: HM2_task5.py [hunk 4]
```diff
@@ -0,0 +1,6 @@
+# 5
+VIP_box = ['Mark', 'Anna', 'Diana', 'Alex', "Inga"]
```
Comment (bohdandovbsyh) [inline HM2_task5.py]: I can change VIP guests. It should be fixed

File: HM2_task6.py [hunk 5]
```diff
@@ -0,0 +1,9 @@
+# 6
+group = ["Jack Pie", "Erik Man", "Lili Evans", "Patrick Low", "Lili Evans", "Patrick Low"]
+uniq_group = []
+for item in group:
+    if item not in uniq_group:
+        uniq_group.append(item)
+print(uniq_group)
```
Comment (bohdandovbsyh) [inline HM2_task6.py]: this HW is for types functions. You can not use loops for solving it

File: HM2_task2.py (hunk 1 above)
Comment (MichaelN98) [reply HM2_task2.py:6]: @bohdandovbsyh, I need to add more employees to "eleks"? I right understand?

Comment (bohdandovbsyh) [reply HM2_task2.py:6]: why add? eleks was taken over by Toshiba. So now eleks do not have employees at all

File: HM2_task6.py [hunk 6]
```diff
@@ -0,0 +1,16 @@
+import numpy as n
+# 6 through numpy, I did not find another option
+group = ["Jack Pie", "Erik Man", "Lili Evans", "Patrick Low", "Lili Evans", "Patrick Low"]
+group = n.unique(group)
+print(group)
```
Comment (bohdandovbsyh) [inline HM2_task6.py:5]: ok. let it be))

another variant is to use fromkeys() function of a dict

**Review by bohdandovbsyh — APPROVED** [review]

File: HM2_task3.py [hunk 7]
```diff
@@ -0,0 +1,22 @@
+# 3
+
+casino_blacklist = ["Jack Pie", "Erik Man", "Lili Evans"]
+poker_blacklist = ["Patrick Low", "Ann Downstown", "Mari Darret"]
+alcohol_blacklist = ["Albert Second", "Kate Signi", "Iri Mane"]
+
+for x in [casino_blacklist, alcohol_blacklist]:
+    poker_blacklist.extend(x)
+
+print(poker_blacklist)
```
Comment (bohdandovbsyh) [inline HM2_task3.py:10]: actually, all these lists are sets because a person can not be in this list twice. 

And the solution is on of the method of sets

**Pull request merged**